### ISSUE TITLE
A new pair of leggings

### ISSUE THREAD

**Issue by Argis13** [Azanor/thaumcraft-suggestions#53]
Every thaumaturge, when not fighting or crafting, wears: 
           Goggles of revealing
            Boots of the traveler
            Thaumostatic Harness 
            Thaumium Fortress Leggings/Void          metal leggings
It may just be my OCD, but I want a pair of leggings to round out this "utility outfit." I was thinking an infusion on a pair of leather pants, with void metal, a portable hole focus, salis mundus

**Comment by Argis13** [comment]
Sorry, tapped on the submit button. Continuing... 
and aer and perdito shards. The essienta cost should be similar to the harness, with very high instability. These "thamual disassociation leggings" make you pass thorough blocks. The downside is, all blocks. You have to wear the Thaumostatic Harness to use them, because all other forms of flying push on air, or push you though air. I think the key binding should be the same as the harness by default, and the research on the eldrich tab.

**Comment by Mechfried** [comment]
I agree a utility leggings would be great, but the proposed ability isn't good since the power will most likely to cause mainly problems

**Comment by Argis13** [comment]
Ok, thought of some "refinements" for this. 
While in "vanish mode:"
You can see through blocks while in them just like in game mode 3
If a player/mob attacks you or you attack another player/mob, it does no damage unless
       A. The weapon is enchanted, in which case only the enchantment damage counts.
       B.  If the weapon is infused, it does full damage, enchantment and all!
You can't see the "real world," only nodes.
If you have a pair of goggles of revealing on, you can see the real world.
It costs as much as the focus costed to activate, and one ordo and one Aer vis per sec. (Edit: Aer vis is not used if have the the vernal heart band sustainer, the logic is that it allows you to breath, and the band does it better.) 
Doesn't work at all the outer lands.
You can only interact with objects (e.g., pick up items, open chests, ect.) while holding an infused object and at the cost of one terra vis.

**Comment by Mechfried** [comment]
Interesting, but would be better was an artifact (like an amulet or a ring) that can't be crafted only found.
For the utility leggings would be a blinking ability much more suited.

**Comment by Argis13** [comment]
Like these ones? #57  Good idea.
